setup.py

```diff
@@ -33,10 +32,10 @@
          ],
     },
     install_requires=[
+        'dash>=1.0.0',
         'bleach>=3.1.0',
         'cryptography>=2.4',
-        'dash==0.41',
-        'dash-auth==1.3.2',
+        'dash-auth>=1.3.2',
         'flask-caching>=1.4.0',
         'flask-talisman>=0.6.0',
         'jinja2>=2.10',
```

Comment:
Consider using this opportunity to change from `>=` to [compatible release (`~=`)](https://www.python.org/dev/peps/pep-0440/#compatible-release).